internal/gengapic/generator.go

```diff
@@ -36,6 +36,12 @@ import (
 	"google.golang.org/protobuf/types/pluginpb"
 )
 
+var allowListEnableNewAuthLibrary = []string{
+	"cloudkms.googleapis.com",
+	"secretmanager.googleapis.com",
+	"showcase.googleapis.com",
+}
+
```

Comment:
I'd just use a `map[string]bool` - no need for custom lookup code.

Reply:
Updated, thanks.